docs/src/main/sphinx/functions/list.rst

```diff
@@ -147,6 +147,7 @@ E
 - :func:`empty_approx_set`
 - ``evaluate_classifier_predictions``
 - :func:`every`
+- :func:`exclude_columns`
```

Comment:
should we have () as well?

Reply:
No, the generate page contains `()`. 
<img width="398" alt="Screenshot 2023-05-10 at 20 21 19" src="https://github.com/trinodb/trino/assets/6237050/926c58df-b78a-44d1-bcbe-1d5264939575">